static/i18n/en-US.json

```
@@ -0,0 +1,547 @@
+  "sort.highest": "Highest Ranking",
+  "sort.alphabetical": "A to Z",
+  "sort.reverse": "Z to A",
+  "layout.list": "List View",
+  "layout.small": "Small View",
+  "layout.detailed": "Detailed View",
+  "layout.pageSize": "Page Size",
+  "pagination.first": "First page",
+  "pagination.previous": "Previous page",
+  "pagination.next": "Next page",
+  "pagination.last": "Last page",
+  "device.by": "by",
```

Biome (1.9.4)

[error] The key device.by was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)